cmd/generate/vault.go

```diff
@@ -39,7 +39,7 @@ func newVaultClient(config vaultClientConfig) (*vaultapi.Client, error) {
 }
 
 func createVaultClientUsingOpsctl(ctx context.Context, gitHubToken, installation string) (*vaultapi.Client, error) {
-	cmdArgs := []string{os.Getenv("SHELL"), "-c", "opsctl create vaultconfig -i " + installation + " -o json"}
```

Comment:
Is there a reason why we called shell with command instead of just calling `opsctl`? It breaks on my machine due to some custom scripting.

Reply:
I was debugging why it wasn't working and that was one of my missed try to fix it.